cmd/agent/app/config.go

```diff
@@ -78,3 +97,65 @@ func requestConfig() (string, error) {
 
 	return string(r), nil
 }
+
+func listRuntimeConfigurableValue(cmd *cobra.Command, args []string) error {
+	err := util.SetAuthToken()
+	if err != nil {
+		return err
+	}
+	c := util.GetClient(false)
+	ipcAddress, err := config.GetIPCAddress()
+	if err != nil {
+		return err
+	}
+	url := fmt.Sprintf("https://%v:%v/agent/config/list-runtime", ipcAddress, config.Datadog.GetInt("cmd_port"))
```

Comment:
Could we set `/agent/config/list-runtime` as a var ?

Reply:
I agree with @remeh, I think it is unlikely that this route will serve elsewhere.

Reply:
moving it to a var is the first step to make sure we won't duplicate it anywhere else.